content/js/record.md

```diff
@@ -1,307 +1,331 @@
 `skygear.privateDB`.
 
 <a name="basic-crud"></a>
 ## Basic CRUD
 
-### Creating a record
+### Creating records
 
-You can save a public record to server as the following.
+Saving a record to the public database:
 
 ``` javascript
-skygear.publicDB.save(new Note({
-  'content': 'Hello World!'
-})).then((record) => {
-  console.log(record);
-}, (error) => {
-  console.error(error);
-});
+const Note = skygear.Record.extend('note');
+const note = new Note({ 'content': 'Hello World!' });
+
+skygear.publicDB.save(note)
+  .then((saved_record) => {
+    console.log(saved_record);
+  })
+  .catch((error) => {
+    console.error(error);
+  });
 ```
 
-#### Batch save
-
-You can also batch save multiple records at one time.
+Saving multiple records to the public database:
 
 ``` javascript
-skygear.publicDb.save([goodNote1, goodNote2, badNote3, goodNote4, badNote5])
-.then((result) => {
-  console.log(result.savedRecords);
-  // [goodNote1, goodNote2, undefined, goodNote4, undefined]
-  console.log(result.errors);
-  // [undefined, undefined, error3, undefined, error5]
-}, (error) => { /* request error */ })
+const note1 = new Note({ date: '2016-09-07', content: 'first note' }),
+const note2 = new Note({ date: '2016-09-08', content: 'second note'}),
+const note3 = new Note({ date: new Date(),   content: 'third note' }),
+const note4 = new Note({ date: '2016-09-10', content: 'forth note' }),
+const note5 = new Note({ date: new Date(),   content: 'fifth note' }),
+
+skygear.publicDB.save([note1, note2, note3, note4, note5])
+  .then((result) => {
+    console.log(result.savedRecords);
+    // [note1, note2, undefined, note4, undefined]
+    console.log(result.errors);
+    // [undefined, undefined, error3, undefined, error5]
+  })
+  .catch((error) => {
+    // request error
+  });
 ```
 
-By default, "good" records are still saved while "bad" records are not.
-Use the `atomic` option if you don't want partial saves, so either all or none
-of the records will be saved.
+By default, only erroneous records are not saved (e.g. due to data type conflict).
+You can use the `atomic` option such that either all or none of the records will be saved:
 
 ``` javascript
-skygear.publicDB.save([goodNote, badNote], { atomic: true });
-// neither of the notes are saved
+skygear.publicDB.save([note1, note2, note3, note4, note5], { atomic: true });
+// none of the records will be saved
 ```
 
-### Reading a record
 
-You can construct a Query object by providing a Record Type.
-You can config the query by mutating its state.
-Read the section about [Query](/js/guide/query) to learn more.
+### Reading records
 
-``` javascript
-const query = new skygear.Query(Blog);
-query.greaterThan('popular', 10);
-query.addDescending('popular');
-query.limit = 10;
-
-skygear.publicDB.query(query).then((records) => {
-  console.log(records)
-}, (error) => {
-  console.error(error);
-})
-```
+Refer to [Query](/js/guide/query) section for details.
```

Comment:
I think it needs a bit more descriptions to make this section sounds natural, because it has a sample query but the reader should go to another page for more info